vm-mock/vm.js

```diff
@@ -104,32 +114,25 @@ export default function createMockedABI(memory, createImports, instantiate, bina
           }
         }
 
-        return webModule.__newArrayBuffer(v);
+        return newArrayBuffer(v);
       },
 
-      // XXX: Untested ABI, directly migrated from as-tester
-
-      // TODO: Please move the ABIs to the previous section once you have confirmed
-      //        that they are functional.
-      // This action can be done on the fly.
-
       assembly_script_reset_storage() {
         resetLedger();
       },
 
       assembly_script_change_call_stack(callstackPtr) {
```

Comment:
I don't find this function in the env, is it outdated ?

Reply:
it's here https://github.com/massalabs/massa-as-sdk/blob/main/assembly/vm-mock/storage.ts#L10

it is used for unit tests of ERC20